### DISCUSSION TITLE
Block unwanted calls from automated calling systems, telemarketers and others.

### DISCUSSION THREAD

**Discussion by Jerzy653** [FossifyOrg/General-Discussion#246] [Ideas]
I am new to this sort of thing.  Firstly, thank you to ALL who make the Fossify Phone app and other open source apps possible. 

This feature screens ALL callers who dial your number, including telemarketers and others. It stops nuisance automated dialer services from completing calls to your phone(s).

When Automated Call Screening is OFF, no filtering of your callers will take place and all calls will ring normally on your phone(s).

When Automated Call Screening is ON and a caller attempts to call your phone number, the caller will be asked to dial a random one digit number. When calls come from an automated service, it is unlikely the service will provide the requested digit and the call will be disconnected or forwarded directly to your voice-mail without ringing; two options. However, if the caller does push the requested digit within the 12 second window, the call will ring normally on your phone(s).

This feature exists within the majicJack VOIP phone where I get zero spam calls.

**Comment by Aga-C** [comment]
Please raise this idea in [Phone](https://github.com/FossifyOrg/Phone/discussions).

**Reply by Jerzy653** [reply]
Thank you; my humble apologies
